mailpoet/lib/Form/Widget.php

```diff
@@ -2,6 +2,9 @@
 
 namespace MailPoet\Form;
 
+if (!defined('ABSPATH')) exit;
+
+
```

Comment:
This is added automatically in the build, and shouldn't be present in the files directly.